### PR TITLE
Xml Dom Mgt.

### PR DESCRIPTION
**Pull request by ModigeMathias**
Codeunit for Xml Dom manipulation
Contains functionality for initializing, reading, and manipulating Xml Document

### CODE REVIEW

File: COD60275.TXT [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      BatchCount@6150629 : Integer;
+      HideDialog@6150621 : Boolean;
+      Initialized@6150622 : Boolean;
+      OutputInitialized@6150631 : Boolean;
+
+    LOCAL PROCEDURE "--- Init Xml"@6014400();
+    BEGIN
+    END;
+
+    PROCEDURE InitDoc@6150639(VAR XmlDoc@6150613 : DotNet "'System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Xml.XmlDocument";RootName@6150615 : Text);
```
Comment (BardurKnudsen) [inline COD60275.TXT]: Either make this procedure local or remove the dotnet types from the parameter list - cannot be called from extensions.

File: COD60275.TXT [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+      OutputOutStr@6150630 : OutStream;
+      ResponseOutStr@6150632 : OutStream;
+      Window@6150615 : Dialog;
+      PrimaryKeyValue@6150616 : Text;
+      BatchCount@6150629 : Integer;
+      HideDialog@6150621 : Boolean;
+      Initialized@6150622 : Boolean;
+      OutputInitialized@6150631 : Boolean;
+
+    LOCAL PROCEDURE "--- Init Xml"@6014400();
```
Comment (BardurKnudsen) [inline COD60275.TXT]: What is this function doing?

File: COD60275.TXT [hunk 3]
```diff
@@ -0,0 +1,260 @@
+OBJECT Codeunit 60275 Xml Dom Mgt.
```
Comment (BardurKnudsen) [inline COD60275.TXT]: Note that we already have codeunit 6224 "XML DOM Management" which seems very similar to this codeunit. Perhaps we are missing some functions in codeunit 6224?

**Comment by ModigeMathias** [comment]
Valid points.
I have hereby changed the codeunit to be focused on Xml Transformation.
If Xml Transformation were to be included in standard either in Codeunit or https://msdn.microsoft.com/en-us/dynamics-nav/developer/devenv-restapi-overview, then this Pull request is obsolete.

**Comment by JesperSchulz** [comment]
We've addressed your requests in pull request #15. We hope that we managed to include all the functions you requested. If we forgot anything, please create a new pull request, based on the submitted code. Thank you very much for your contributions! Your added functions will be released with the November update.

**Pull request closed without merging**